pakyow-support/lib/pakyow/support/defineable.rb

```diff
@@ -100,6 +122,12 @@ def stateful(name, object)
         def define(&block)
           instance_eval(&block)
         end
+
+        # Provide a default make method
+        #
+        def make(*args, &block)
+          new(*args, &block)
```

Comment:
I was thinking that the default make method would create an instance and then instance_eval the block.

```ruby
instance = self.new(*args)
instance.instance_eval(&block)
instance
```

Is there a reason it shouldn't work this way?